### DISCUSSION TITLE
[Request: gameplay overlay] Add a "rolling" and maybe more animations to score and combo counter.

### DISCUSSION THREAD

**Discussion by Nope10** [ppy/osu#24749] [Ideas]
Hello, i've got a request regarding the **score and combo counter**, i was wondering if it could be possible to get new animations style for those counters that we could change in a drop down menu located in right part of the skin / gameplay overlay editor (Shift + F2).

As i just have one animation idea which would be a "rolling up" animation (like in the slots machine) i was wondering if with my post, you could eventually think about giving a "fresh" look to the score and combo counter and giving the choice to players to chose either default animation or a roll animation.

This idea came from the CounterStrike 2 new in-game UI, when you earn money there's a rolling up animation and i thought it could be cool to have this kind of animations for osu! score/combo overlay, it would add more customisation possibilites for the skins.

**Comment by peppy** [comment]
Most counters already have rolling animations. Which are you specifically talking about?

**Reply by ItsShamed** [reply]
I think they want something like [this](https://github.hubspot.com/odometer/docs/welcome/) (this is the library that most if not all gosumemory overlays use for their counters).

**Reply by Nope10** [reply]
Sorry for the late answerm at first I wanted to do a default animation and the roll animation but yes, as ItsShamed said, it's exactly this kind of animation I was trying to describe, it's not a huge feature but I think having different animations for counters could be nice for the skin customization.

I guess it's possible to work on it through Rulesets but sadly i'm not a dev and even though my PC is a gaming computer, it struggles to run Visual Studio (and I don't have a lot of disk space too).
